LocalSettings.php

```diff
@@ -501,6 +501,12 @@
 		'default' => true,
 	],
 
+	// Display Title
+	'wgDisplayTitleHideSubtitle' => [
+		'default' => false,
+		'solarawiki' => true,
```

Comment:
You can remove this, then do it in Special:ManageWiki/settings when merged

Reply:
Bah, of course!